test/safe_float_multiplication_test.cpp

```diff
@@ -59,11 +59,23 @@ BOOST_AUTO_TEST_CASE_TEMPLATE( safe_float_multiplication_throws_on_overflow, FPT
 
 BOOST_AUTO_TEST_CASE_TEMPLATE( safe_float_multiplication_inexact_rounding, FPT, test_types){
     // define two FPT numbers suppose to produce inexact rounding
-    FPT a = 2 * (pow(2, std::numeric_limits<FPT>::digits)-1);
-    FPT b = pow(2, std::numeric_limits<FPT>::digits)-1;
+    FPT a, b;
+    
+    if(std::is_same<FPT, float>()) {
+        a = 1.64005529880523681640625f;
```

Comment:
The reason to construct using functions and limit values is because these numbers can vary between platforms. However it is safe to use then like this until we start testing in a platform not paying the tests.